### ISSUE TITLE
EXTRACTJSONFIELD JSON array support

### ISSUE THREAD

**Issue by joewood** [confluentinc/ksql#523]
It seems that the EXTRACTJSONFIELD function doesn't support array extraction, for example the following doesn't work:
```
EXTRACTJSONFIELD(myfield,'$[0].subfield')
```

When `myfield` is `[ { "subfield":"blah"} ]`

**Comment by satybald** [comment]
@joewood you need extract object in the column i.e. myfield[0]. 

```
EXTRACTJSONFIELD(myfield[0],'$subfield')
```

**Comment by apurvam** [comment]
Just merged #61. Closing this out.
